### PR TITLE
Added -daemon option in chef-client service install

### PR DESCRIPTION
**Pull request by Vasu1105**
This PR will make the Azure Chef extension install the service by passing a flag to the MSI instead of duplicating the code to create the service.

### CODE REVIEW

File: lib/chef/azure/service.rb [hunk 1]
```diff
@@ -154,10 +156,9 @@ def disable_cron
   end
 
   ## enable/install chef-service on Windows platform ##
-  def enable_service(bootstrap_directory, log_location)
+  def install_service
```
Comment (btm) [inline lib/chef/azure/service.rb]: Do we still need to install the service here? Does this happen via the MSI now?

Comment (btm) [inline lib/chef/azure/service.rb]: We can leave as is for now.

**Comment by Vasu1105** [comment]
@NimishaS  & @btm hold on for this PR to get merge. Found one issue.

**Comment by Vasu1105** [comment]
@NimishaS & @btm I have fixed the error.

**Pull request merged**